pkg/devicetwin/dtcontext/dtcontext_test.go

```diff
@@ -0,0 +1,451 @@
+package dtcontext
+
+import (
+	"reflect"
+	"sync"
+	"testing"
+
+	"github.com/kubeedge/kubeedge/beehive/pkg/common/config"
+	"github.com/kubeedge/kubeedge/beehive/pkg/core/context"
+	"github.com/kubeedge/kubeedge/beehive/pkg/core/model"
+	"github.com/kubeedge/kubeedge/pkg/common/util"
+	"github.com/kubeedge/kubeedge/pkg/devicetwin/dtcommon"
+	"github.com/kubeedge/kubeedge/pkg/devicetwin/dttype"
+)
+
+// TestInitDTContext is function to test InitDTContext().
+func TestInitDTContext(t *testing.T) {
+	util.LoadConfig()
+	nodeID, err := config.CONFIG.GetValue("edgehub.controller.node-id").ToString()
+	if err != nil {
+		t.Errorf("Error in getting node id %v", err)
+		return
+	}
+	tests := []struct {
+		name    string
+		context *context.Context
+		want    *DTContext
+		wantErr error
+	}{
+		{
+			name:    "ActualContextArgumentTest",
+			context: context.GetContext(context.MsgCtxTypeChannel),
+			want: &DTContext{
+				GroupID:        "",
```

Comment:
string dataType has default value `""`. I guess no need for this line.
Applicable to all similar cases.